bin/pycbc_optimize_snr

```diff
@@ -250,7 +250,7 @@ for ifo in ifos:
     snr_slice = snr_series_dict[ifo].time_slice(time_window[0], time_window[1])
     max_snr_idx = numpy.argmax(abs(snr_slice))
     idx_offset = snr_slice.start_time - snr_series_dict[ifo].start_time
-    idx_offset = int(idx_offset * sample_rate + 0.5)
```

Comment:
It's not clear to me why this would change anything ... but I won't object.

Reply:
Apparently `idx_offset` is a `LIGOTimeGPS` and it does not like to be multiplied by a float (I get the "argument 2 of type double" exception).

Reply:
Ah, right okay.